pom.xml

```diff
@@ -393,7 +393,7 @@
     <!-- replaces findbugs-maven-plugin -->
     <version.com.github.spotbugs-maven-plugin>3.1.8</version.com.github.spotbugs-maven-plugin>
 
-    <maven.min.version>3.5.2</maven.min.version>
+    <maven.min.version>3.6.3</maven.min.version>
```

Comment:
kogito-runtimes requires maven minimum of 3.6.2 ( > 3.5.3, but < 3.6.3) - that version likely fixes the optaplanner repo issues too.
https://github.com/kiegroup/kogito-runtimes/blob/master/pom.xml#L71

Still, let's just switch to 3.6.3, the latest and greatest.

Reply:
both of them are available https://github.com/kiegroup/kie-jenkins-scripts/blob/f9aa5f70936b2b884154c0864fa5ad67e36b86ed/jenkins-slaves/ansible/kie-rhel7.yml#L7